src/untangled/client/core.cljc

```diff
@@ -78,8 +78,11 @@
   (map->Application {:initial-state      initial-state
                      :mutation-merge     mutation-merge
                      :started-callback   started-callback
-                     :reconciler-options (merge {:migrate migrate :pathopt pathopt :shared shared}
-                                           reconciler-options)
+                     :reconciler-options (merge (cond-> {}
+                                                  migrate (assoc :migrate migrate)
+                                                  pathopt (assoc :pathopt pathopt)
+                                                  shared (assoc :shared shared))
+                                                reconciler-options)
```

Comment:
Bummer. That's unfortunate, eh? Oh well, I see nothing else to do that doesn't break compatibility.